src/apollo/mutations/mutations.js

```diff
@@ -0,0 +1,112 @@
+import {
+  accountsReactive,
+  networkIdReactive,
+  networkReactive,
+  reverseRecordReactive,
+  isReadOnlyReactive,
+  isRunningAsSafeAppReactive,
+  favouritesReactive,
+  subDomainFavouritesReactive,
+  web3ProviderReactive
+} from '../reactiveVars'
+import { getAccounts, getNetwork, getNetworkId } from '@ensdomains/ui'
+import { disconnect, connect } from '../../api/web3modal'
+import { getReverseRecord } from '../sideEffects'
+import { isRunningAsSafeApp } from 'utils/safeApps'
+
+export const setWeb3ProviderLocalMutation = async provider => {
+  web3ProviderReactive(provider)
+
+  const accounts = await getAccounts()
+
+  if (provider) {
+    provider.removeAllListeners()
+    // setNetworkIdLocalMutation(provider.network.chainId)
+    setAccountsLocalMutation(accounts)
+  }
+
+  provider?.on('chainChanged', _chainId => {
+    console.log('chain changed: ', _chainId)
+    setNetworkIdLocalMutation(parseInt(_chainId))
+    // getNetworkMutation()
+  })
+
+  provider?.on('accountsChanged', async accounts => {
+    console.log('accounts changed')
+    // setAccountsLocalMutation(accounts)
```

Comment:
This function now doesn't do anything. If not needed, can you take it out?

Reply:
> Looks like many CI tests have been failing https://app.travis-ci.com/github/ensdomains/ens-app/jobs/536569396 Maybe these tests are a bit fragile?

mmm that's weird they were passing 2 days ago, but yes in general i've been struggling with the cypress tests